requirements/prod.in

```diff
@@ -9,7 +9,7 @@ pyramid-sanity
 # 3rd-party packages
 python-jose[cryptography]
 PyJWT
-SQLAlchemy >= 1.1.4
```

Comment:
We want to upgrade soon but I it doesn't need to happen at the same exact time. 

Pinning to what we actually support now.